app/src/org/commcare/preferences/HiddenPreferences.java

```diff
@@ -96,6 +96,7 @@ public class HiddenPreferences {
 
 
     private static final long NO_OF_HOURS_TO_WAIT_TO_RESUME_BACKGROUND_WORK = 36;
+    private static final String ENABLE_SECURE_FLAG = "cc-enable-secure-flag";
```

Comment:
Can we name it `cc-enable-secure-app-window` instead to imply it's usage better.

Reply:
+1 that if we aren't going to wrap the functionality behind a more generic preference of bundled features (or cross platform features) we should articulate its purpose as specifically as possible (IE: "cc-enable-android-window-secure-flag") to prevent people from misinterpreting it as more powerful or more important than it is in practice.